Cannot Deploy ; TypeError: localErrorHandlerData.handle is not a function when updated to 2.69.0 version

### Are you certain it's a bug?

- [X] Yes, it looks like a bug

### Is the issue caused by a plugin?

- [X] It is not a plugin issue

### Are you using the latest version?

- [X] Yes, I'm using the latest version

### Is there an existing issue for this?

- [X] I have searched existing issues, it hasn't been reported yet

### Issue description

When running serverless deploy through CLI, I get the below error. 

```
TypeError: localErrorHandlerData.handle is not a function
    at module.exports (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/lib/cli/handle-error.js:109:29)
    at process.<anonymous> (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/scripts/serverless.js:53:3)
    at Object.onceWrapper (events.js:422:26)
    at process.emit (events.js:327:22)
    at process.EventEmitter.emit (domain.js:483:12)
    at process._fatalException (internal/process/execution.js:165:25)
```

### Service configuration (serverless.yml) content

```yaml
# This file is the main config file for the service.
# For full config options, check the docs:
#    docs.serverless.com

plugins:
  - serverless-plugin-stage-variables
  - serverless-plugin-test-helper
  - serverless-plugin-aws-alerts
  - serverless-plugin-split-stacks
  - serverless-create-dynamodb-global-tables-for-cf-stack
  - serverless-prune-plugin
  - serverless-sqs-alarms-plugin
  - serverless-plugin-conditional-functions
  - serverless-plugin-resource-tagging
  - serverless-plugin-log-retention
  - serverless-plugin-lambda-dead-letter

provider:
  name: aws
  runtime: nodejs12.x

package:
  excludeDevDependencies: false
  individually: true
  include:
    - '!./**'

functions:  #Adding random function name since we have around 30 functions 
login:
    handler: handler/handler.login
choose:
    handler: handler/handler.assign_role
request:
    handler: handler/student_handler.request
```


### Command name and used flags

serverless deploy --region us-east-1 --stage test1

### Command output

```shell
2021-12-15T07:23:30.8259612Z Serverless: Running "serverless" installed locally (in service node_modules)
2021-12-15T07:23:31.8567640Z Serverless: To ensure safe major version upgrades ensure "frameworkVersion" setting in service configuration (recommended setup: "frameworkVersion: ^2.69.0")
2021-12-15T07:23:31.8568257Z 
2021-12-15T07:23:32.4466190Z Serverless: Load command interactiveCli
2021-12-15T07:23:32.4484820Z Serverless: Load command config
2021-12-15T07:23:32.4486091Z Serverless: Load command config:credentials
2021-12-15T07:23:32.4486985Z Serverless: Load command config:tabcompletion
2021-12-15T07:23:32.4487523Z Serverless: Load command config:tabcompletion:install
2021-12-15T07:23:32.4488046Z Serverless: Load command config:tabcompletion:uninstall
2021-12-15T07:23:32.4496286Z Serverless: Load command create
2021-12-15T07:23:32.4500042Z Serverless: Load command install
2021-12-15T07:23:32.4504502Z Serverless: Load command package
2021-12-15T07:23:32.4510579Z Serverless: Load command deploy
2021-12-15T07:23:32.4511683Z Serverless: Load command deploy:function
2021-12-15T07:23:32.4512328Z Serverless: Load command deploy:list
2021-12-15T07:23:32.4512864Z Serverless: Load command deploy:list:functions
2021-12-15T07:23:32.4523373Z Serverless: Load command invoke
2021-12-15T07:23:32.4523844Z Serverless: Load command invoke:local
2021-12-15T07:23:32.4530821Z Serverless: Load command info
2021-12-15T07:23:32.4533984Z Serverless: Load command logs
2021-12-15T07:23:32.4537606Z Serverless: Load command metrics
2021-12-15T07:23:32.4541435Z Serverless: Load command print
2021-12-15T07:23:32.4545384Z Serverless: Load command remove
2021-12-15T07:23:32.4549109Z Serverless: Load command rollback
2021-12-15T07:23:32.4549595Z Serverless: Load command rollback:function
2021-12-15T07:23:32.4554637Z Serverless: Load command slstats
2021-12-15T07:23:32.4557219Z Serverless: Load command plugin
2021-12-15T07:23:32.4560247Z Serverless: Load command plugin
2021-12-15T07:23:32.4560758Z Serverless: Load command plugin:install
2021-12-15T07:23:32.4564890Z Serverless: Load command plugin
2021-12-15T07:23:32.4565413Z Serverless: Load command plugin:uninstall
2021-12-15T07:23:32.4569038Z Serverless: Load command plugin
2021-12-15T07:23:32.4569515Z Serverless: Load command plugin:list
2021-12-15T07:23:32.4572726Z Serverless: Load command plugin
2021-12-15T07:23:32.4573322Z Serverless: Load command plugin:search
2021-12-15T07:23:32.4576681Z Serverless: Load command config
2021-12-15T07:23:32.4577171Z Serverless: Load command config:credentials
2021-12-15T07:23:32.4762010Z Serverless: Load command upgrade
2021-12-15T07:23:32.4763525Z Serverless: Load command uninstall
2021-12-15T07:23:33.0197319Z Serverless: Load command prune
2021-12-15T07:23:33.0218856Z Serverless: WARNING: Plugin Plugin uses deprecated hook deploy:compileEvents,
2021-12-15T07:23:33.0219839Z                      use package:compileEvents hook instead
2021-12-15T07:23:33.0228254Z Serverless: Load command login
2021-12-15T07:23:33.0229628Z Serverless: Load command logout
2021-12-15T07:23:33.0231392Z Serverless: Load command generate-event
2021-12-15T07:23:33.0232229Z Serverless: Load command test
2021-12-15T07:23:33.0235077Z Serverless: Load command dashboard
2021-12-15T07:23:33.0237488Z Serverless: Load command output
2021-12-15T07:23:33.0237948Z Serverless: Load command output:get
2021-12-15T07:23:33.0238414Z Serverless: Load command output:list
2021-12-15T07:23:33.0241976Z Serverless: Load command param
2021-12-15T07:23:33.0242414Z Serverless: Load command param:get
2021-12-15T07:23:33.0243123Z Serverless: Load command param:list
2021-12-15T07:23:33.0245851Z Serverless: Load command studio
2021-12-15T07:23:33.2072110Z Serverless: [AWS sts 200 0.16s 0 retries] getCallerIdentity({})
2021-12-15T07:23:33.2122523Z Serverless: Skipping variables resolution with old resolver (new resolver reported no more variables to resolve)
2021-12-15T07:23:33.9288743Z Serverless:  
2021-12-15T07:23:33.9289515Z Serverless: Learn more about configuration validation here: http://slss.io/configuration-validation
2021-12-15T07:23:33.9290044Z Serverless:  
2021-12-15T07:23:33.9395940Z Serverless: Deprecation warning: Resolution of lambda version hashes was improved with better algorithm, which will be used in next major release.
2021-12-15T07:23:33.9396670Z             Switch to it now by setting "provider.lambdaHashingVersion" to "20201221".
2021-12-15T07:23:33.9397379Z             While it is highly encouraged to upgrade to new algorithm, you can still use the old approach by setting "provider.lambdaHashingVersion" to "20200924".
2021-12-15T07:23:33.9398133Z             More Info: https://www.serverless.com/framework/docs/deprecations/#LAMBDA_HASHING_VERSION_V2
2021-12-15T07:23:33.9412872Z Serverless: Invoke deploy
2021-12-15T07:23:33.9416812Z Serverless: Invoke package
2021-12-15T07:23:33.9419262Z Serverless: Invoke aws:common:validate
2021-12-15T07:23:33.9421127Z Serverless: Invoke aws:common:cleanupTempDir
2021-12-15T07:23:33.9500822Z Serverless: Packaging service...
2021-12-15T07:23:34.1724419Z Serverless: [AWS sts 200 0.242s 0 retries] getCallerIdentity({})
2021-12-15T07:24:46.5034295Z /root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/node_modules/essentials/index.js:12
2021-12-15T07:24:46.5035348Z 		throw reason;
2021-12-15T07:24:46.5036053Z 		^
2021-12-15T07:24:46.5036871Z 
2021-12-15T07:24:46.5037864Z TypeError: localErrorHandlerData.handle is not a function
2021-12-15T07:24:46.5039373Z     at module.exports (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/lib/cli/handle-error.js:109:29)
2021-12-15T07:24:46.5040153Z     at process.<anonymous> (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/scripts/serverless.js:53:3)
2021-12-15T07:24:46.5040883Z     at Object.onceWrapper (events.js:422:26)
2021-12-15T07:24:46.5041437Z     at process.emit (events.js:327:22)
2021-12-15T07:24:46.5041950Z     at process.EventEmitter.emit (domain.js:483:12)
2021-12-15T07:24:46.5042506Z     at process._fatalException (internal/process/execution.js:165:25)
2021-12-15T07:24:46.6421578Z ##[error]Bash exited with code '1'.
2021-12-15T07:24:46.6435168Z ##[error]Bash wrote one or more lines to the standard error stream.
2021-12-15T07:24:46.6439068Z ##[error]/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/node_modules/essentials/index.js:12
		throw reason;
		^

TypeError: localErrorHandlerData.handle is not a function
    at module.exports (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/lib/cli/handle-error.js:109:29)
    at process.<anonymous> (/root/agent01/_work/_tool/node/12.18.3/x64/lib/node_modules/serverless/scripts/serverless.js:53:3)
    at Object.onceWrapper (events.js:422:26)
    at process.emit (events.js:327:22)
    at process.EventEmitter.emit (domain.js:483:12)
    at process._fatalException (internal/process/execution.js:165:25)
```


### Environment information

```shell
Serverless: Running "serverless" installed locally (in service node_modules)
Framework Core: 2.69.0 (local)
Plugin: 5.5.1
SDK: 4.3.0
Components: 3.18.1
```

--- a/lib/cli/handle-error.js
+++ b/lib/cli/handle-error.js
@@ -4,6 +4,7 @@ const path = require('path');
 const isObject = require('type/object/is');
 const chalk = require('chalk');
 const stripAnsi = require('strip-ansi');
+const _ = require('lodash');
 const sfeVersion = require('@serverless/dashboard-plugin/package.json').version;
 const { platformClientVersion } = require('@serverless/dashboard-plugin');
 const { getDashboardProvidersUrl } = require('@serverless/dashboard-plugin/lib/dashboard');
@@ -105,7 +106,7 @@ module.exports = async (exception, options = {}) => {
       }
     })();
 
-    if (localErrorHandlerData) {
+    if (_.get(localErrorHandlerData, 'handle')) {
       localErrorHandlerData.handle(exception, localErrorHandlerData.options);
       return;
     }

